.rubocop.yml

```diff
@@ -61,6 +61,11 @@ Style/CollectionMethods:
 Style/WordArray:
   Enabled: false
 
+# in rspec, expect{...} is conventional
```

Comment:
Yep, keep the spaces until we have a discussion about changing the agreed upon style.  I'm personally happy with the existing convention.